Adds Initial Set of Documents (#3)
ArlingtonCouncilOfPTAs/tech-guide: google-workspaces.md

# Google Workspaces for Non-Profits
Google offers [Workspace for Non-Profits](https://www.google.com/nonprofits/), which is a free version of it's business product.  This product includes accounts for users, email, drive storage, mailing lists, chat, and more.

## Sign Up (Non-Profit)
Before you can sign up for Google Workspace for Non-Profits, you will need to have a personal Google account to make the request.

> Wait until [TechSoup](https://www.techsoup.org) shows that your [account status](https://www.techsoup.org/my-account) is "Qualified.".  You will need your validation token to proceed.

Reviewer: ```suggestion
Wait until [TechSoup](https://www.techsoup.org) shows that your [account status](https://www.techsoup.org/my-account) is **"Qualified."**.  You will need your validation token to proceed.
```

If you do not want to wait, you can still proceed with the instructions in the next section: Sign Up (Business).  If you do that, don't forget to come back and upgrade to a Google Workspace for Non-Profits account before your free trial ends.

1. Click the Get Started button on the [Google for Non-Profits](https://www.google.com/nonprofits/) page.

> 2. You will need to be logged in as your own personal account (@gmail.com).  If you have a Google account through another organization, verify you are not logged in to that account.

Reviewer: ```suggestion
2. You will need to be logged in as your own personal account (@gmail.com). 
- **NOTE**: If you have a Google account through another organization, verify you are not logged in to that account.
```

3. Click Next through the screens, reading and answering the questions.
4. When prompted, indicate YES that you do have a TechSoup account.

> 5. Continue through the steps until you are finished.

Author: I need to update these steps whenever I do the next org setup.

Author: Non-blocking, documented in issue #11

## Sign Up (Business)

> Follow these instructions if you don't have access to your [tax documents](./tax-documents.md), or there is a problem with your [Tech Soup](./techsoup.md) validation.  After your free trial, you will be charged $12/mo/user if you follow these instructions, but you can upgrade to [Workspace for Non-Profits](https://www.google.com/nonprofits/) later, once you get your paper work in order.

Reviewer: ```suggestion
Follow these instructions if you don't have access to your [tax documents](./tax-documents.md), or if there is a problem with your [Tech Soup](./techsoup.md) validation.  After your free trial, you will be charged $12/mo/user, but you can upgrade to [Workspace for Non-Profits](https://www.google.com/nonprofits/) later, once you get your paperwork in order.
```

1. Click the Get Started button on the [Google Workspace](https://workspace.google.com/) page.

> 2. Enter your own personal name, email address, and phone number.

Reviewer: ```suggestion
2. Enter your personal name, email address, and phone number.
```

3. If your PTA has a domain name, enter it when prompted.
4. Enter your new username (email address) and password when prompted.

> 5. A message will appear informing you that Business Standard costs $12/mo/user, but you will get a 14 day free trial.  You can downgrade to Business Starter ($6/mo/user) from the admin panel at any time.

Reviewer: ```suggestion
5. A message will appear informing you that "Business Standard" costs $12/mo/user, but you will get a 14 day free trial.  You can downgrade to Business Starter ($6/mo/user) from the admin panel at any time.
```

6. Click CONTINUE when you are done entering users, and you will be taken to a page where you can email those users their new login information.
7. For each user, click the mail icon.  Enter their personal email address and click SEND.  When you are done, click CONTINUE.
8. On the final step, click ACTIVATE to enable email services on your account.
9. Read the messages, and click I'M READY TO ACTIVATE GMAIL.

At this point, you should have working email for yourself and the users you created.  Users also have access to the standard suite of Google Workspace features such as Drive.  If you wish to create an email list, you can do so using the instructions in the next section of this guide.

## Add an Email Group
Use these instructions if you wish to create an email list that goes out to multiple users.

1. Login to the [Admin Panel](https://admin.google.com).
2. Click on Directory, then Groups.
3. Click the "Create group" link.
4. Enter a name and email address for the group.  It is not necessary to enter any group owners.

> 5. Leave the Security checkbox unchecked.

Reviewer: ```suggestion
5. Leave the Security checkbox UNCHECKED.
```

6. Click the NEXT button.
7. In the permissions table, ensure that all of the boxes are checked in the "Who can post" row.  See the included screenshot below for reference.  (Double check this.  Skipping this step is a common mistake.)
8. In the "Who can join this group" section, select "Only invited members."
9. Click the CREATE GROUP button.
10. On the "What you can do next" page, click the "Add members" link.  This takes you to the group management page.
11. Click the "Add members" link.
12. Enter an email address and then click the ADD TO GROUP button.
13. Repeat steps 11-13 for each user you want to add to the group.

> [<- back](./README.md)

Reviewer: ```suggestion
[<- Go Back](./README.md)
```
